Header header
geometry_msgs/Pose2D robotPose
float64[] landmarkX
float64[] landmarkY
float64[] landmarkr
